hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DistributedFileSystem.java

```diff
@@ -3913,15 +3913,14 @@ public Path doCall(final Path p) throws IOException {
       @Override
       public Path next(final FileSystem fs, final Path p)
           throws IOException {
-        if (fs instanceof DistributedFileSystem) {
-          DistributedFileSystem myDfs = (DistributedFileSystem) fs;
-          return myDfs.getEnclosingRoot(p);
-        } else {
+        if (!(fs instanceof DistributedFileSystem)) {
           throw new UnsupportedOperationException(
-              "Cannot call getEZForPath"
+              "Cannot call getEnclosingRoot"
```

Comment:
does this still hold?

Reply:
removed the whole block since this exists in base Filesystem impl